src/plugins/screenshot_mode/common/get_set_browser_screenshot_mode.ts

```diff
@@ -15,9 +15,9 @@ export const KBN_SCREENSHOT_MODE_ENABLED_KEY = '__KBN_SCREENSHOT_MODE_ENABLED_KE
  * localStorage. The ability to set a value in localStorage enables more convenient development and testing
  * in functionality that needs to detect screenshot mode.
  */
-export const getScreenshotMode = (): unknown => {
+export const getScreenshotMode = (): boolean => {
   return (
-    ((window as unknown) as Record<string, unknown>)[KBN_SCREENSHOT_MODE_ENABLED_KEY] ||
+    Boolean(((window as unknown) as Record<string, unknown>)[KBN_SCREENSHOT_MODE_ENABLED_KEY]) ||
```

Comment:
Happy to update this function to return a boolean!

However, I'd prefer this code to be explicit about the value it expects to find at `KBN_SCREENSHOT_MODE_ENABLED_KEY` since this forms part of the contract. I'll update on my PR with my take!